### PR TITLE
Add warning message if min chrome version check fails

### PR DESCRIPTION
**Pull request by beaufortfrancois**
Hello Jeff,
I believe the simple warning added in this CL will help developers who want to try out our samples but are not on Canary for instance.

screenshot 2016-06-15 at 11 18 13 am

R=@jeffposnick

### CODE REVIEW

**Comment by jeffposnick** [comment]
My preference would be not to go this route. While I understand that this code will only run if it thinks the user agent is already (and older) Chrome, I don't like the message that the sample *requires" Chrome, since ideally we're only writing samples for standard features that will eventually make it to all browsers.

I also am worried that there are samples out there that are targeted to version X, but have either a polyfill or a workaround in place to show how they could be used in version < X, and we'd end up scaring developers away with that status.

**Comment by beaufortfrancois** [comment]
> My preference would be not to go this route. While I understand that this code will only run if it thinks the user agent is already (and older) Chrome, I don't like the message that the sample *requires" Chrome, since ideally we're only writing samples for standard features that will eventually make it to all browsers.

How about rewording the message to something like `Warning! Keep in mind this sample has been tested with Chrome 54.` ?

> I also am worried that there are samples out there that are targeted to version X, but have either a polyfill or a workaround in place to show how they could be used in version < X, and we'd end up scaring developers away with that status.

**Comment by jeffposnick** [comment]
I don't think that would address my second concern, which is that some samples are written to explicitly illustrate backwards compatibility. For example, https://github.com/GoogleChrome/samples/blob/gh-pages/idb-getall/demo.js#L46

If you'd like to add this into a template that's specific to the Bluetooth samples, where you know the backwards compatibility story is unlikely to change, then I'm cool with that.

**Comment by beaufortfrancois** [comment]
I see... ;(
I think I'll add a boolean setting that each sample can define to display a warning if min chrome version is not met yet.

**Comment by beaufortfrancois** [comment]
Hey @jeffposnick What do you think of https://github.com/GoogleChrome/samples/pull/369/commits/bfc3dcde62235fd5362be6e95f7a1be067960c93?

**Comment by jeffposnick** [comment]
👍

**Comment by Psoffie009** [comment]
> Hello Jeff,
> I believe the simple warning added in this CL will help developers who want to try out our samples but are not on Canary for instance.
> 
> screenshot 2016-06-15 at 11 18 13 am
> 
> R=@jeffposnick

**Comment by Psoffie009** [comment]
> My preference would be not to go this route. While I understand that this code will only run if it thinks the user agent is already (and older) Chrome, I don't like the message that the sample *requires" Chrome, since ideally we're only writing samples for standard features that will eventually make it to all browsers.
> 
> I also am worried that there are samples out there that are targeted to version X, but have either a polyfill or a workaround in place to show how they could be used in version < X, and we'd end up scaring developers away with that status.
